.gitignore

```diff
@@ -7,4 +7,5 @@ __pycache__
 *.pyc
 pi_config.ini
 lambda/buttons-lambda.zip
-lambda/package
\ No newline at end of file
+lambda/package
+.DS_Store
```

Comment:
Noting here that I added .DS_Store to the .gitignore. These files are generated in almost every directory on MacOS, so I want to prevent it accidentally getting added by future committers.

I am also doing this so my `git status` commands are cleaner after creating commits.